fe/fe-core/src/main/java/com/starrocks/connector/kudu/KuduPredicateConverter.java

```diff
@@ -192,7 +189,8 @@ private Object getLiteral(ScalarOperator operator) {
                 return (int) ChronoUnit.DAYS.between(epochDay, localDate);
             case DATETIME:
                 LocalDateTime localDateTime = constValue.getDatetime();
-                return fromSQLTimestamp(Timestamp.valueOf((localDateTime)));
+                LocalDateTime epochDateTime = Instant.ofEpochSecond(0).atOffset(ZoneOffset.UTC).toLocalDateTime();
```

Comment:
does the timestamp have a timezone?

Reply:
no, the timestamp does not include timezone information.